blog/fine-grained-news-2024-11.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Fine Grained News - November 2024
+description: Fine Grained News
+slug: fine-grained-news-2024-11
+date: 2024-11-30
+authors: hello-caleb
+tags: [newsletter]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# Fine Grained News - November 2024
+
+Welcome to the November edition of Fine Grained News! As we enter the final stretch of 2024, there are exciting developments in the OpenFGA to share.
+
+🌟 **We just hit 3,000 stars on the OpenFGA repo!** 🌟 Because of this great community, we've just this incredible milestone! Thank you so much for all the support you've shown this project. Let's keep the momentum going! If you haven't yet, we'd greatly appreciate you [starring the repo](https://github.com/openfga/openfga) to help push us toward 4,000 stars and grow our amazing community! 
+
+![Celebrating OpenFGA reaching 3,000 GitHub stars](../static/img/blog/fgn-2024-11-stars.png)
+
+## Just Shipped
+
+**v1.8.1:** This release focuses on performance improvements and enhanced monitoring capabilities, including two new flags for better control over check operations, improved performance for TTU relationships with set operations, and expanded metrics tracking with duration measurements. The release also adds deduplication logic to BatchCheck API and introduces new logging fields for authz calls. [Read more in the Changelog...](https://github.com/openfga/openfga/compare/v1.8.0...v1.8.1)
+
+For more about the new OPENFGA_CHECK_ITERATOR_TTL and OPENFGA_CHECK_CACHE_LIMIT flags, run `./openfga run --help`  
+
+**Batch Check API:**  We've launched Batch Check API in OpenFGA v1.8.0. Batching authorization checks together in a single request will significantly reduce roundtrip network latency. In addition, v.1.8.0 also added support for Contextual Tuples in the `Expand` API, and included other performance improvements. [Read more in the Changelog...](https://github.com/openfga/openfga/compare/v1.7.0...v1.8.0)
+
+## **Coming Up**
+
+**SDK Updates:** We will be updating the SDKs next to take advantage of the new BatchCheck, starting  with Python and JavaScript. If you want to see an SDK prioritized, let us know!
+**SDK Updates:** We will be updating the SDKs next to take advantage of the new BatchCheck, starting  with Python and JavaScript. If you want to see an SDK prioritized, let us know!
```

Comment:
This line is repeated - should something else be here instead?

Reply:
Maybe the work on the Changelog filter on the SDKs?

Reply:
```suggestion
To clarify, were you thinking the changelog filtering work on the `ReadChanges` API that was done in v1.8.0? If so, that maybe better to work into the paragraph on Batch Check since it was the same release.

How about editing the Batch Check paragraph on L.25 like this: 

**Batch Check API:**  We've launched Batch Check API in OpenFGA v1.8.0. Batching authorization checks together in a single request will significantly reduce roundtrip network latency. In addition, v.1.8.0 also added support for Contextual Tuples in the `Expand` API, time-based filtering in the `ReadChanges` API, and other performance improvements. Read more in the Changelog...
```suggestion
```

Reply:
I went ahead and added time filtering to the additional update included in v1.8.0 outside of Batch Check.